### PR TITLE
Add logic for user survey notification

### PR DESCRIPTION
**Pull request by kmjennison**

### CODE REVIEW

**Comment by kmjennison** [comment]
Bug identified: Only render the notification on the client side. Otherwise, it will render server-side even after it's been dismissed.

**Review by jedialkimiya — APPROVED** [review]

File: src/pages/index.js [hunk 1]
```diff
@@ -540,73 +580,116 @@ const Index = ({ data: fallbackData }) => {
...
-                      <SettingsIcon
-                        className={clsx(
-                          classes.userMenuItem,
-                          classes.settingsIcon
-                        )}
-                      />
-                    </IconButton>
-                  </Link>
+                {/**
+                 * TODO: consolidate all notifications here to manage
```
Comment (jedialkimiya) [inline src/pages/index.js:654]: What would be the ideal way to have another notification here? Asking for Search test. Just put it under and have it display below in the y-position?

**Review by jedtan — APPROVED** [review]

File: src/pages/index.js [hunk 2]
```diff
@@ -540,73 +580,116 @@ const Index = ({ data: fallbackData }) => {
...
+                 * visible state. Right now, these will overlay the ones
+                 * that appear via the UserImpact component.
+                 */}
+                <div className={classes.notificationsContainer}>
+                  {showUserSurvey ? (
+                    <Notification
```
Comment (jedtan) [inline src/pages/index.js:660]: Is the idea to add further notifications below in the y-position?

**Comment by kmjennison** [comment]
Yeah, let's do that for now.

**Pull request merged**